cni/pkg/install/install_test.go

```diff
@@ -221,7 +220,7 @@ func TestSleepCheckInstall(t *testing.T) {
 
 			select {
 			case <-readyChan:
-				assert.Truef(t, isReady.Load().(bool), "isReady should have been set to true")
+				assert.Equal(t, isReady.Load(), true)
```

Comment:
Nit: why remove the message "isReady should have been set to true"?

Reply:
This message doesn't have any value. Equal will already emit that its 'false' if the test case fails and the line that fails.